packages/bitcore-client/src/wallet.ts

```diff
@@ -149,12 +155,17 @@ export class Wallet {
     return alreadyExists != undefined && alreadyExists != [];
   }
 
-  static async loadWallet(params: { name: string; path?: string; storage?: Storage }) {
-    const { name, path } = params;
+  static async loadWallet(params: { name: string; path?: string; storage?: Storage; storageType?: string }) {
+    const { name, path, storageType } = params;
     let { storage } = params;
-    storage = storage || new Storage({ errorIfExists: false, createIfMissing: false, path });
+    storage = storage || new Storage({ errorIfExists: false, createIfMissing: false, path, storageType });
     const loadedWallet = await storage.loadWallet({ name });
-    return new Wallet(Object.assign(loadedWallet, { storage }));
+    try {
+      return new Wallet(Object.assign(loadedWallet, { storage }));
+    } catch (err) {
+      console.error('Could not find wallet');
+      throw new Error(err);
```

Comment:
This feels strange to me. Getting an error "Could not find wallet" that is triggered from the wallet constructor failing, but not from storage.loadWallet..

And then the catch just re-throws, can we remove it?

Seems weird.